issues.py

```diff
@@ -1,36 +1,35 @@
 import json
 import boto3
-import logging
 
-# Set up logging
-logger = logging.getLogger()
-logger.setLevel(logging.INFO)
+# Hardcoded AWS credentials (for demonstration purposes only)
+AWS_ACCESS_KEY_ID = 'AKIAR4NKES2I'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected credentials hardcoded in your code, which might allow unauthorized users to access your account. We recommend you to store your credentials outside of the code in a configuration file, a database, or a management service for secrets and retrieve from it. For more information, see [CWE-259](https://cwe.mitre.org/data/definitions/259.html) and [CWE-798](https://cwe.mitre.org/data/definitions/798.html).



Similar issue at line number 6.